governance/api.go

```diff
@@ -114,13 +109,13 @@ func (api *PublicGovernanceAPI) Vote(key string, val interface{}) (string, error
 }
 
 func (api *PublicGovernanceAPI) isRemovingSelf(val interface{}) bool {
```

Comment:
If you always receive string here, what about changing of the parameter type to string instead of interface?

Reply:
Of you need to check once again to avoid panic here.
```go
s, ok := val.(string)
```

Reply:
Yes, it always receives `string` because non-string is handled by the above `ValidateVote`. 
Reflected at https://github.com/klaytn/klaytn/pull/1110/commits/4caf232a66b2d7eaaa5b94a02ef1fd127745dc6e.